.gitignore

```diff
@@ -29,9 +29,8 @@ workspace.xml
 # Build script
 build.sh
 
-# Test java and python files
-src/main/java/CompSci162.java
-src/main/java/SearchingMethods.java
+# Test java and python files/packages
+src/main/java/playground
```

Comment:
[multi-module](https://docs.gradle.org/current/userguide/multi_project_builds.html) projects may also make sense, but it may also be overkill